lib/jormungandr/src/Cardano/Wallet/Jormungandr/Environment.hs

```diff
@@ -75,8 +75,8 @@ instance ToText Network where
 instance KnownNetwork 'Mainnet where
     networkVal = Mainnet
     hrp = unsafeHumanReadablePart "ca"
-    single = 0x03
-    grouped = 0x04
+    single = 0x3
+    grouped = 0x4
```

Comment:
Where does this comes from? 

--> https://github.com/input-output-hk/implementation-decisions/blob/master/text/0001-address.md#binary-serialization

I think the values were already right.

Reply:
https://github.com/input-output-hk/rust-cardano/blob/e5d974f7bedeb00c9c9d688ac66094a34bf8f40d/chain-addr/src/lib.rs#L95-L98

Yeah, should maybe link it in the code

Reply:
I mean, that's the same thing in the end. But I don't think the change is necessary ^^"

Reply:
`0x3 = 3 = 0x03`

Reply:
I do think `0x03` reads better however and is slightly less confusing..

Reply:
😂 🤦‍♂